device_test.go

```diff
@@ -0,0 +1,30 @@
+package gotorch_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	torch "github.com/wangkuiyi/gotorch"
+)
+
+func TestDevice(t *testing.T) {
+	a := assert.New(t)
+	var device torch.Device
+	if torch.IsCUDAAvailable() {
+		device = torch.NewDevice("cuda")
+	} else {
+		device = torch.NewDevice("cpu")
+	}
+	a.NotNil(device)
+	x := torch.RandN([]int64{2, 3}, false)
+	x.To(device)
```

Comment:
Should we add [`assert.NotPanics`](https://godoc.org/github.com/stretchr/testify/assert#NotPanics) check here?